obspy/clients/eida/__init__.py

```diff
@@ -10,120 +13,182 @@
 :license:
     GNU Lesser General Public License, Version 3
     (https://www.gnu.org/copyleft/lesser.html)
+
+Basic Usage
+-----------
+
+The first step is always to initialize a client object.
+
+>>> from obspy.clients.eida import Client
+>>> client = Client("GFZ")
+
+A client object can be initialized either with the base URL of any EIDA routing
+service or with a shortcut name that will be mapped to such a URL. The examples
```

Comment:
> Not sure how you want this exactly to be typeset. Like: :mod:`a shortcut name that will be mapped to such a URL <obspy.clients.fdsn>`

Something like that, yes. That would make it a link to FDSN module API page (with the short URLS displayed in the example) with link text "a shortcut name that will be mapped to such a URL".

> Note that not all shortcut names work, only those that run the routing service. Except when using the even service that is not routed.

Maybe add a note in brackets..   ".. (only works if node supports EIDA routing)" ?
In any case, I think it should somehow be clear what short URLs can be used and expected to work. Also, will it show a meaningful message if a not-supporting node is targetted by the client initialization?

> Also, some doctest fails because request to 5 nodes (default) is made in parallel and the order of networks in print(inventory) depends on the order of responses received. We cannot rely on station descriptions not changing either.

You could sort networks by code on the resulting inventory before returning it with the following call, might be good anyway to be as deterministic as possible with the result:

```python
inv.networks = sorted(inv.networks, key=lambda x: x.code)
```